lib/mssql.js

```diff
@@ -2,13 +2,12 @@ import Ajv from "ajv";
 import JSONStream from "JSONStream";
 import {json} from "micro";
 import mssql from "mssql";
-import {failedCheck, badRequest, notImplemented} from "./errors.js";
 import {Transform} from "stream";
 
-const TYPES = mssql.TYPES;
-
-const pools = new Map();
+import {failedCheck, badRequest, notImplemented} from "./errors.js";
+import pools, {keyOf} from "./pools.js";
```

Comment:
keyOf isn't referenced.
```suggestion
import pools from "./pools.js";
```